collect.py

```diff
@@ -218,24 +219,36 @@ def run_test(bug, browser, driver, op_sequence=None):
     return saved_sequence
 
 
+def read_sequence(bug_id):
+    with open('data/%d.txt' % bug_id) as f:
```

Comment:
Wrap this in a try except (only catch the exception thrown when the file doesn't exist). In the except case, return an empty list.